benchmarks/load_test/README.md

```diff
@@ -0,0 +1,22 @@
+### Load testing
+
+Create a Locust load testing instance by using the CLI tool `load_test.py`
+
+It *requires* locust to be installed, use `pip install locustio` (if dev-dependencies is not already installed).
+
+#### Usage
+
+- Start port-forwarding with *ambassador* on a cluster (with an existing gordo-project).
+
+    `kubectl port-forward svc/ambassador -n ambassador 8888:80`
+
+- Use the CLI tool `load_test.py`, specify the *project name* (required), *host* (optional) and *port* (optional).
+***Note** the port needs to match the port which is port-forwarded to*
+
+- Go to: [locust instance](http://127.0.0.1:8089) and specify the number of users to simulate, as well as the the number of users
```

Comment:
I like the link, but should perhaps be [locust instance: http://127.0.0.1:8089](http://127.0.0.1:8089) just so it's explicit _where_ the default server is.